src/com/carolinarollergirls/scoreboard/core/Rulesets.java

```diff
@@ -43,14 +44,13 @@
     public Ruleset addRuleset(String name, String parentId, String id);
 
     public enum Value implements PermanentProperty {
-	ID,
-	NAME,
-	RULE_DEFINITIONS,
-	RULE,
-	RULESET;
+	CURRENT_RULESET_ID,
+	CURRENT_RULESET_NAME;
     }
     public enum Child implements AddRemoveProperty {
-	KNOWN_RULESETS;
```

Comment:
One of the reasons I went with a Known prefix is so that you won't pull in all the rulesets if you just care about the current settings. I think that's still the case here.